examples/hacker_news_assets/hacker_news_assets_tests/test_resources/test_parquet_io_manager.py

```diff
@@ -26,21 +26,20 @@ def spark_input_asset(pandas_df_asset: SparkDF):
         return pandas_df_asset
 
     with tempfile.TemporaryDirectory() as temp_dir:
-        io_manager_test_job = build_assets_job(
-            "io_manager_test_job",
+        res = materialize_to_memory(
             assets=[pandas_df_asset, spark_input_asset],
-            resource_defs={
+            resources={
                 "pyspark": pyspark_resource,
                 "io_manager": local_partitioned_parquet_io_manager.configured(
                     {"base_path": temp_dir}
                 ),
             },
+            partition_key="2022-01-01-16:00",
         )
 
         expected_path = os.path.join(
             temp_dir, "pandas_df_asset", "20220101160000_20220101170000.pq"
         )
-        res = io_manager_test_job.execute_in_process(partition_key="2022-01-01-16:00")
         assert res.success
         assert os.path.exists(expected_path)
```

Comment:
Does this work? If we're invoking `materialize_to_memory` here, I would think that would either ignore the IO manager or error?